.github/workflows/acr-build-publish.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           elif [ ${{ matrix.components}} == "devhostagent.restorationjob" ]; then
             echo "tag=$MINDARO_DEVHOSTAGENT_RESTORATIONJOB_TAG" >> $GITHUB_ENV
           else
-            echo "tag=$MINDARO_ROUTINGMANAGER_TAG" >> $GITHUB_ENV
+            echo "tag=stable" >> $GITHUB_ENV
```

Comment:
The code always uses stable tag instead of the tag from imagetag.settings file just for routing manager component.